public/pages/SplitIndex/container/SplitIndex/SplitIndex.tsx

```diff
@@ -269,5 +270,7 @@ export class SplitIndex extends Component<SplitIndexProps> {
 export default function SplitIndexWrapper(props: Omit<SplitIndexProps, "commonService" | "coreService">) {
   const services = useContext(ServicesContext) as BrowserServices;
   const coreService = useContext(CoreServicesContext) as CoreStart;
+  // in split-index we don't change the data source picker
```

Comment:
Could you please elaborate more on this part? I get a little bit confused by the comment and the code in this line.

Reply:
In split index, dataSourcePicker is in read-only mode..